### PR TITLE
VS 2017 sln

### PR DESCRIPTION
**Pull request by lzybkr**
I created a VS2017 solution file for the powershell-win-core csproj and everything it references.

It "upgraded" each csproj - but I reverted all changes but the one to suppress future upgrades.

The solution doesn't build and I haven't investigated why, but it's useful for editing across the entire project.

### CODE REVIEW

**Comment by iSazonov** [comment]
Maybe put the file in root folder with PowerShell.Common.props file? Currently I use @powercode sln file such way.

**Comment by lzybkr** [comment]
I'll defer to @daxian-dbw on taking the sln from @powercode or this one. I can move it to the root if we decide to take this.

File: src/powershell-win-core/powershell-win-core.csproj [hunk 1]
```diff
@@ -55,5 +53,7 @@
   <PropertyGroup Condition=" '$(Configuration)' == 'CodeCoverage' ">
     <DebugType>full</DebugType>
   </PropertyGroup>
-
+  <PropertyGroup>
+    <RootNamespace>Microsoft.PowerShell</RootNamespace>
```
Comment (daxian-dbw) [inline src/powershell-win-core/powershell-win-core.csproj]: Is the <RootNamespace> necessary to be added?
If so, can you please move it to the `PropertyGroup` section at top of the file?

**Review by daxian-dbw — CHANGES_REQUESTED** [review]

**Comment by daxian-dbw** [comment]
I will take this `.sln` file since we already have a PR.
@lzybkr please move the `.sln` to the root and rename it to `PowerShell-Win.sln`. VS2017 for Mac is available, so maybe someone will create a `PowerShell-Mac.sln` for that.

**Comment by daxian-dbw** [comment]
@lzybkr Could you please address the comments when you have time? Thanks!

File: src/powershell-win-core/powershell-win-core.csproj (hunk 1 above)
Comment (lzybkr) [reply src/powershell-win-core/powershell-win-core.csproj]: It avoids a warning when opening the solution. I moved it to the top.

**Comment by lzybkr** [comment]
Comments addressed and I rebased on master to fix the conflict.

**Review by daxian-dbw — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**